lm75bd/lm75bd.c

```diff
@@ -26,7 +26,26 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
 }
 
 error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {
-  /* Implement this driver function */
+  error_code_t errCode;
+
+  if (temp == NULL) return ERR_CODE_INVALID_ARG;
+
+  uint8_t ptrByte = 0x00;
```

Comment:
I think it's more clear to just make 2 buffers using uint8_t writeByte[1] = {0} (or just one buffer with length 2)